src/lando/main/auth.py

```diff
@@ -73,6 +77,30 @@ def wrapper(request, *args, **kwargs):
     return wrapper
 
 
+def force_auth_refresh(f: Callable) -> Callable:
+    """
+    Decorator which forces authenticated session to be refreshed.
+    """
+
+    def wrapper(*args, **kwargs):
+        """Set oidc_id_token_expiration to 0, forcing session refresh."""
+        # First check that SessionRefresh is indeed enabled.
+        if "mozilla_django_oidc.middleware.SessionRefresh" not in settings.MIDDLEWARE:
+            raise RuntimeError("SessionRefresh middleware required but is not enabled.")
+
+        # Find the request in the arguments. This is needed in case this decorator
+        # is used in class-based views. The request would be the first argument that
+        # is a WSGIRequest instance.
+        request = [arg for arg in args if isinstance(arg, WSGIRequest)][0]
```

Comment:
Is the arg not in the same position for different views?

Reply:
Django has both class based views, and method based views. This ensures things work for both types of views. For the former, the request is in the second argument.